stacker_blueprints/elasticache/base.py

```diff
@@ -222,6 +222,11 @@ def create_replication_group(self):
         maintenance_window = variables["PreferredMaintenanceWindow"] or \
             NOVALUE
 
+        if snapshot_window == NOVALUE:
```

Comment:
I think it'd probably be better to throw an error if the `snapshot_window` is set to `NOVALUE`, and `snapshot_retention_limit` is set to something else.  Better to let people know their config is broken, rather than changing expectations behind the scenes.